website_cookieconsent/static/src/js/website_cookieconsent.js

```diff
@@ -0,0 +1,68 @@
+$(function() {
+    "use strict";
```

Comment:
I think instead of using this start, and then call `$.ajax` from the main, the following is more in line with Odoo:

https://github.com/OCA/website/blob/8.0/website_backend_views/static/src/js/website_backend_views.js#L23

Reply:
This should be the one for website js in v 8.0